tests/test.c

```diff
@@ -402,7 +402,7 @@ struct test tests[] = {
         _TEST(0110_batch_size, 0),
         _TEST(0111_delay_create_topics, 0, TEST_BRKVER_TOPIC_ADMINAPI,
               .scenario = "noautocreate"),
-        _TEST(0112_cooperative_rebalance, 0),
+        _TEST(0112_cooperative_rebalance, 0, TEST_F_LOCAL, TEST_BRKVER(2,4,0,0)),
```

Comment:
TEST_F_LOCAL means it does not need a cluster, which conflicts with specifying a minimum broker version with TEST_BRKVER. which is it? :)